proposals/kroxylicious-operator-api-v1alpha.md

```diff
@@ -1,7 +1,7 @@
-# Kroxylicious Operator API - Alternative Proposal.
+# Kroxylicious Operator API - v1alpha
```

Comment:
Do we intend to reflect the actually implemented API back to this repo? 
If so should this be `v1alpha1` 
if not should this be `v1alpha skeleton`  or `v1alpha proposal`?

Reply:
> Do we intend to reflect the actually implemented API back to this repo?

I don't think we should do that.  I think our CRDs should become the source of truth.  

I think updating the proposal to assist our development is fine.